src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -282,10 +288,18 @@ export class ExtensionEditor extends BaseEditor {
 			})
 			.then(null, () => {
 				const p = append(this.content, $('p'));
-				p.textContent = localize('noReadme', "No README available.");
+				p.textContent = noContentCopy;
 			}));
 	}
 
+	private openReadme(extension: IExtension) {
+		return this.openMarkdown(extension, this.extensionReadme.get(), localize('noReadme', "No README available."));
```

Comment:
I do not know how expansive is to call `localize`
If it's a lot, we could probably use `localize.bind(this, 'noReadme', "No README available.")` and use the new returned function only if it's necessary. What's your opinion?